app.rb

```diff
@@ -0,0 +1,97 @@
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+  end
+
+  def list_all_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}"
+    end
+  end
+
+  def create_a_teacher
+    puts 'enter your name'
+    name = gets.chomp
+    puts 'enter your age'
+    age = gets.chomp.to_i
+    puts 'enter your specialization'
+    specialization = gets.chomp
+    teacher = Teacher.new(name, age, specialization)
+    @people << teacher
+    puts 'Teacher created successfully.'
+  end
+
+  def create_a_student
+    puts 'enter your name'
+    name = gets.chomp
+    puts 'enter your age'
+    age = gets.chomp.to_i
+    puts 'what is the parent_permission? (Y/N)'
+    parent_permission = gets.chomp
+
+    student = Student.new(name, age, parent_permission)
+    @people << student
+    puts 'Student created successfully.'
+  end
+
+  def create_a_book
+    puts 'enter the title of the book'
+    title = gets.chomp
+    puts 'enter the author'
+    author = gets.chomp
+
+    book = Book.new(title, author)
+    @books << book
+    puts 'Book created successfully.'
+  end
+
+  def create_a_rental
+    puts 'Choose a book from the list by number'
+    @books.each_with_index do |book, index|
+      puts "#{index} - #{book.title} by #{book.author}"
+    end
+    book_index = gets.chomp.to_i
+    book = @books[book_index]
+    puts 'Choose a person from the list by number:'
+    @people.each_with_index do |person, index|
+      puts "#{index} - #{person.name} - #{person.age} - #{person.class}"
```

Comment:
- [ ] According to the wireframe you need to return the people in this format. Kindly apply it. 
![image](https://github.com/vanina123/School-libary/assets/91486791/b3c81421-4ee8-463a-878b-eead1097c4b2)
your app:
![image](https://github.com/vanina123/School-libary/assets/91486791/eb00eb12-5e1f-409c-a628-25e4428b2df2)